packages/react-native-audio-api/src/core/AudioScheduledSourceNode.ts

```diff
@@ -40,6 +38,7 @@ export default class AudioScheduledSourceNode extends AudioNode {
       );
     }
 
+    // TODO: should we reset hasBeenStarted here? :thunk:
```

Comment:
No, because source can be started only once, so we want to remember if this source already has been started